packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -66,10 +66,19 @@ const PersonalDetailsForm = props => {
     }, [should_close_tooltip, handleToolTipStatus, setShouldCloseTooltip]);
 
     React.useEffect(() => {
+        if (
+            !no_confirmation_needed &&
+            typeof status === 'object' &&
+            !values.confirmation_checkbox &&
+            is_qualified_for_idv
+        ) {
+            setStatus({ ...status, is_confirmed: false });
+        }
         if (no_confirmation_needed && typeof status === 'object' && !status.is_confirmed) {
             setStatus({ ...status, is_confirmed: true });
         }
-    }, [no_confirmation_needed, setStatus, status]);
+        //eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [no_confirmation_needed]);
```

Comment:
Han e refactor this in a. better way rather than removing the dependency?

Reply:
yes, will refactor the logic